internal/config/config.go

```diff
@@ -76,12 +79,22 @@ func Load(path string) (*Config, error) {
 
 	//TODO ensure values of the variables are checked for nil before passing down to docker.
 
+	tag := ""
+	out, err := exec.Command("git", "rev-parse", "--short", "HEAD").Output()
+	if err != nil {
+		pterm.Warning.Println("could not run git rev-parse, the default tag was set")
```

Comment:
Should we also say which tag was set? 
I'd also reorder the message and setting a tag, as technically we haven't set it yet.